cryptographic_estimators/PKEstimator/pk_problem.py

```diff
@@ -29,9 +29,9 @@ class PKProblem(BaseProblem):
     - ``n`` -- columns of the matrix
     - ``m`` -- rows of the matrix
     - ``q`` -- size of the field
-    - ``ell`` -- rows of the matrix whose permutation should lie in the kernel
-    - ``use_parity_row`` -- enables trick of appending extra (all one) row to the matrix, i.e., m -> m+1 (default:False)
-    - ``nsolutions`` -- number of (expected) solutions of the problem in logarithmic scale
+    - ``ell`` -- number of rows of the matrix whose permutation should lie in the kernel (default: 1)
+    - ``use_parity_row`` -- enables trick of appending extra (all one) row to the matrix, i.e., m -> m+1 (default: false)
+    - ``nsolutions`` -- number of solutions of the problem in logarithmic scale (default: expected_number_solutions)
```

Comment:
I just realized, that this is not entirely true, as the default is the maximum between the  expected number and 1 (or 0 in log scale)